tests/transaction.js

```diff
@@ -1,4 +1,7 @@
-const Transaction  = require('../transaction');
+const fs = require('fs');
+const path = require('path');
+
+const Transaction = require('../transaction');
```

Comment:
This doesn't return a class (the class is in `exports.Transaction`), so don't capitalize.

Reply:
The diff is showing this as a change, but that require statement was already there. I can update it, but it would introduce churn that wasn't required for this fix.

Let me know if you want me to introduce changes to correct lint warnings, even for parts of the files that were not related/modified by this PR.

Reply:
Oh yeah ignore me then.

Reply:
It's just a test file. Churn implies no improvement, and I think we can all agree that non-class variables shouldn't be capitalized. I'd prefer it was corrected.